generator/src/test/java/com/scottlogic/deg/generator/generation/UpfrontTreePrunerTests.java

```diff
@@ -64,7 +64,7 @@ void runUpfrontPrune_withOneField_returnsPrunedTree() {
             ConstraintNode unPrunedRoot = Mockito.mock(ConstraintNode.class);
             DecisionTree tree = new DecisionTree(unPrunedRoot, new ProfileFields(fields));
             DecisionTree treeMarkedWithContradictions = new DecisionTree(
-                new ConstraintNode().markNode(NodeMarking.CONTRADICTORY),
+                new com.scottlogic.deg.generator.decisiontree.ConstraintNodeBuilder().build().builder().markNode(NodeMarking.CONTRADICTORY).build(),
```

Comment:
Import the builder

Reply:
Yeah looks I forgot to change this in a few places. I think I have caught them all